.travis.yml

```diff
@@ -1,12 +1,25 @@
 language: elixir
 sudo: false
 elixir:
-  - 1.4
-  - 1.5
-  - 1.6
+  - 1.7
+  - 1.8
+  - 1.9
+  - 1.10
 otp_release:
   - 19.3
   - 20.0
+  - 21.0
+  - 22.0
+matrix:
```

Comment:
Should we add Elixir 1.10/OTP 21 and Elixir 1.10/OTP 22 to the matrix as well?

Reply:
Good news! The matrix defined here is what will be `exclude`d from the combinator of `elixir` and `otp_release` versions which are run, so both [OTP 21](https://travis-ci.org/github/duffelhq/paginator/jobs/718733690) and [OTP 22](https://travis-ci.org/github/duffelhq/paginator/jobs/718733691) will already be run for Elixir 1.10.

Reply:
Apologies - could have spotted that myself. Thanks for clarifying.

Reply:
No worries, it's easy to misinterpret.